server/org.eclipse.glsp.example.workflow/src/main/java/org/eclipse/glsp/example/workflow/handler/CreateActivityNodeHandler.java

```diff
@@ -27,12 +27,16 @@
 public abstract class CreateActivityNodeHandler extends CreateNodeOperationHandler {
 
    private final String label;
+   private final String elementTypeId;
 
    public CreateActivityNodeHandler(final String elementTypeId, final String label) {
       super(elementTypeId);
+      this.elementTypeId = elementTypeId;
       this.label = label;
    }
 
+   protected String getElementTypeId() { return elementTypeId; }
```

Comment:
Shouldn't we push this into `CreateNodeOperationHandler`?

Reply:
And, if you agree, also for `CreateEdgeOperationHandler` I guess...

Reply:
I'm not sure if this is generally applicable. In the case of the example specific handlers we know that we always have one element type id and never change this via setter. This assumption does not hold for the generic handlers

Reply:
But this catched another issue. I forgot to add the new constructor for multiple element type to `CreateNodeOpreationHandler` and `CreateEdgeOperationHandler`. I will create a follow up PR for this